### ISSUE TITLE
Duplicate indexes

### ISSUE THREAD

**Issue by chomudda** [willbryant/kitchen_sync#53]
KS tries to create multiple indexes when syncing from MySQL to PostgreSQL

```
CREATE INDEX "room_info_id_idx" ON library_config ("room_info_id");
CREATE INDEX "room_info_id_idx" ON notification_info ("room_info_id");
CREATE INDEX "room_info_id_idx" ON position_data ("room_info_id");
CREATE INDEX "room_info_id_idx" ON vital_info ("room_info_id");
```

Which returns the following error:

`ERROR:  relation "room_info_id_idx" already exists`

**Comment by chomudda** [comment]
Fixed the error myself, MySQL allows duplicate index names, however, PostgreSQL does not.

**Comment by ashwinimone** [comment]
How did you fix this error?

**Comment by willbryant** [comment]
I believe you would need to rename the index to be unique across the schema in order to get it into pg.

**Comment by willbryant** [comment]
Implemented automatic renaming inside KS under #95.
